pkg/runner/http.go

```diff
@@ -258,6 +260,11 @@ func (r *simpleTestCaseRunner) GetSuggestedAPIs(suite *testing.TestSuite, api st
 
 func (r *simpleTestCaseRunner) withResponseRecord(resp *http.Response) (responseBodyData []byte, err error) {
 	responseBodyData, err = io.ReadAll(resp.Body)
+	if len(responseBodyData) > DefaultMaxRecvMsgSize {
+		err = fmt.Errorf("HTTP response body is too large")
+		return
+	}
+
```

Comment:
Actually, HTTP response does not have this limit. The `gRPC` do. I prefer to set the limit in the `gRPC` response function. For example:

https://github.com/LinuxSuRen/api-testing/blob/e9974374b116dc0f7de540027410b3f2350bf7a5/pkg/server/remote_server.go#L444